tests/utils/TestnetProcedures.sol

```diff
@@ -490,4 +492,110 @@ contract TestnetProcedures is Test, DeployUtils, FfiUtils, DefaultMarketInput {
         })
       );
   }
+
+  function _upgradeToStandardAToken(address asset, string memory symbol) internal {
+    (address aToken, , ) = contracts.protocolDataProvider.getReserveTokensAddresses(asset);
+    uint256 currentRevision;
+    try MockRwaATokenInstance(aToken).getMockRevision() returns (uint256 revision) {
+      currentRevision = revision;
+    } catch {
+      currentRevision = 1;
+    }
+
+    MockATokenInstance mockAToken = new MockATokenInstance(
+      IPool(report.poolProxy),
+      currentRevision + 1
+    );
+
+    vm.prank(poolAdmin);
+    contracts.poolConfiguratorProxy.updateAToken(
+      ConfiguratorInputTypes.UpdateATokenInput({
+        asset: asset,
+        treasury: report.treasury,
+        incentivesController: report.rewardsControllerProxy,
+        name: symbol,
+        symbol: symbol,
+        implementation: address(mockAToken),
+        params: abi.encode()
+      })
+    );
+    vm.stopPrank();
+  }
+
+  function _upgradeToRwaAToken(address asset, string memory symbol) internal {
+    (address aToken, , ) = contracts.protocolDataProvider.getReserveTokensAddresses(asset);
+    uint256 currentRevision;
+    try MockRwaATokenInstance(aToken).getMockRevision() returns (uint256 revision) {
```

Comment:
```suggestion
    try MockATokenInstance(aToken).getMockRevision() returns (uint256 revision) {
```

Reply:
ATOKEN_REVISION will always return 1. The mocks are mocking `getRevision`, not the ATOKEN_REVISION constant